troi/playlist.py

```diff
@@ -255,6 +284,11 @@ def submit_to_spotify(self, user_id: str, token: str, is_public: bool = True, is
             result = sp.playlist_add_items(spotify_playlist["id"], spotify_track_ids)
             submitted.append((spotify_playlist["external_urls"]["spotify"], spotify_playlist["id"]))
 
+            if playlist.external_urls:
```

Comment:
Need to decide on exact way how this should be stored in Playlist entity. We also likely want to store the unmatched tracks or least their count.